carbon/src/main/res/values/attrs.xml

```diff
@@ -482,15 +482,16 @@
         <attr name="carbon_insetBottom"/>
         <attr name="carbon_insetColor"/>
 
+        <attr name="carbon_stroke" />
```

Comment:
pretty much all views have stroke attrs after cornerRadius. I don't think that this refactor is neccessary

Reply:
@ZieIony got your idea